### PR TITLE
[DOCS-2986] Update CWS rules doc to reflect rule split

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Updates the CWS rules documentation to reflect upcoming changes, which includes a new Agent Configuration tab in Setup and Configuration.

### Motivation
Jira request

### Preview
https://docs-staging.datadoghq.com/sarina/cws-rule-split/security_platform/cloud_workload_security/workload_security_rules?tab=host

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 1]
```diff
@@ -134,19 +155,21 @@ Complete the remaining steps based on your environment:
...
-10. Upgrade the helm chart with `helm upgrade <RELEASENAME> -f values.yaml --set datadog.apiKey=<APIKEY> datadog/datadog`.
+3. Upgrade the helm chart with `helm upgrade <RELEASENAME> -f values.yaml --set datadog.apiKey=<APIKEY> datadog/datadog`.
 
     **Note:** If you need to make further changes to `default.policy`, you can either use `kubectl edit cm jdefaultpol` or replace the configMap with  `kubectl create configmap jdefaultpol --from-file default.policy -o yaml --dry-run=client | kubectl replace -f -`.
 
+
 {{% /tab %}}
 {{< /tabs >}}
 
+To finalize setup, restart the [Datadog Agent][5].
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:166]:
```suggestion
To finalize your setup, restart the [Datadog Agent][5].
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 2]
```diff
@@ -69,52 +73,69 @@ For example, if you used explicit filenames to evaluate open events (for example
...
 
-4. Enter the rule in the **Expression** field using [Datadog Security Language (SECL)][5]. A rule can have multiple Agent expressions combined with boolean logic (`||`, `&&`). You can also change the **counter**, **group by**, and **roll-up** window.
+6. Click Download Workload Security Policy in the top right corner to download a default policy file to your local machine.
+
+### Configure the rule
+
+Once your new default policy file is downloaded, navigate to the [**Rules** page][3].
+
+1. Click the **New Rule** button in the top right.
+2. Select **Workload Security** under **Rule types**. Select a detection method, such as **Threshold** or **New Term**.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:108]:
```suggestion
2. Select **Workload Security** under **Rule types**. Select a detection method such as **Threshold** or **New Term**.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 3]
```diff
@@ -69,52 +73,69 @@ For example, if you used explicit filenames to evaluate open events (for example
...
+    ```text
+    exec.file.path in ["/usr/bin/docker", "/usr/local/bin/docker",
+    "/usr/bin/kubectl", "/usr/local/bin/kubectl"] && container.id != ""
+    ```
 
-    {{< img src="security_platform/cws/workload_security_rules/create_cws_rule.png" alt="Configuring a new Cloud Workload Security rule with rule type as Workload Security" >}}
+5. Save the rule. This automatically navigates you back to the Rules page.
 
-4. Enter the rule in the **Expression** field using [Datadog Security Language (SECL)][5]. A rule can have multiple Agent expressions combined with boolean logic (`||`, `&&`). You can also change the **counter**, **group by**, and **roll-up** window.
+6. Click Download Workload Security Policy in the top right corner to download a default policy file to your local machine.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:101]:
```suggestion
6. Click **Download Workload Security Policy** in the top right corner to download a default policy file to your local machine.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 4]
```diff
@@ -69,52 +73,69 @@ For example, if you used explicit filenames to evaluate open events (for example
...
+
+    For example, to monitor for suspicious container clients:
+
+    ```text
+    exec.file.path in ["/usr/bin/docker", "/usr/local/bin/docker",
+    "/usr/bin/kubectl", "/usr/local/bin/kubectl"] && container.id != ""
+    ```
 
-    {{< img src="security_platform/cws/workload_security_rules/create_cws_rule.png" alt="Configuring a new Cloud Workload Security rule with rule type as Workload Security" >}}
+5. Save the rule. This automatically navigates you back to the Rules page.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:99]:
```suggestion
5. Save the rule. This automatically navigates you back to the **Rules** page.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 5]
```diff
@@ -69,52 +73,69 @@ For example, if you used explicit filenames to evaluate open events (for example
...
-To manually add or update rules, load a new `default.policy` file to the Agent.
+2. Click **Add an Agent Rule** in the top right.
 
-Follow the instructions below:
+3. Add a name and description for the rule.
 
-1. In Datadog, navigate to the [**Rules** page][4].
-2. Click **New Rule** in the top right.
-3. Select **Workload Security** under **Rule types**.
+4. Define the Agent expression. Enter the rule in the Expression field using Datadog Security Language (SECL).
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:88]:
```suggestion
4. Define the Agent expression by entering the rule in the **Expression** field with Datadog Security Language (SECL).
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 6]
```diff
@@ -69,52 +73,69 @@ For example, if you used explicit filenames to evaluate open events (for example
...
+
+### Create a new default policy file
+
+First, create a new default policy file to load to the Agent by following the instructions below.
 
-To receive the latest bundled CWS rules, [upgrade the Datadog Agent][3].
+1. In Datadog, navigate to the [Agent Configuration page][4] under Setup and Configuration.
 
-To manually add or update rules, load a new `default.policy` file to the Agent.
+2. Click **Add an Agent Rule** in the top right.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:84]:
```suggestion
2. Click **Add an Agent Rule** in the top right.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 7]
```diff
@@ -69,52 +73,69 @@ For example, if you used explicit filenames to evaluate open events (for example
...
 
-## Add and update rules
+## Setup and configuration
+
+### Create a new default policy file
+
+First, create a new default policy file to load to the Agent by following the instructions below.
 
-To receive the latest bundled CWS rules, [upgrade the Datadog Agent][3].
+1. In Datadog, navigate to the [Agent Configuration page][4] under Setup and Configuration.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:82]:
```suggestion
1. In Datadog, navigate to the [Agent Configuration page][4] under **Setup & Configuration**.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 8]
```diff
@@ -69,52 +73,69 @@ For example, if you used explicit filenames to evaluate open events (for example
 
 Approvers are generally more powerful and preferred. Using approvers, the Agent can process only what it needs to see rather than dynamically learning what to filter out.
 
-## Add and update rules
+## Setup and configuration
+
+### Create a new default policy file
+
+First, create a new default policy file to load to the Agent by following the instructions below.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:80]: "New" is redundant when we talk about creating (new) things.

```suggestion
### Create a default policy file

First, create a default policy file to load to the Agent by following the instructions below.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 9]
```diff
@@ -53,11 +57,11 @@ To ensure that your policy is evaluated in-kernel for maximum efficiency, always
 - `[event_type].file.name`
 - `[event_type].file.path`
 
-`[event_type]` can be open or exec, for example.
+**Note**: `[event_type]` can be open or exec, for example.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:60]: Do you think "exec" as an executable command should be in monospace here? Feel free to discard this suggestion.

```suggestion
**Note**: `[event_type]` can be open or `exec`, for example.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 10]
```diff
@@ -15,23 +15,23 @@ further_reading:
...
+With Cloud Workload Security (CWS) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of rules to detect suspicious behavior. Datadog provides [bundled CWS Agent rules][1] that you receive when you upgrade the Datadog Agent. However, you can write your own custom Agent rules. This guide covers the concept of Agent rules and provides instruction on how to create them.
 
-The full set of rules for the Agent is called a policy. Datadog provides several [out-of-the-box Cloud Workload Security rules][2] for your convenience in the default policy.
+### Agent rules
 
-At some point, you will likely want to write your own custom rules for the Agent to use. Below are example rules, guidelines for optimization, and the underlying mechanics of rule efficiency.
+An Agent Rule contains the [Agent Expression][2], which determines which activity to collect from the Agent. These collected events are then evaluated based on patterns of events described in a [rule][3]. A full set of rules for the Agent is called a policy. Datadog provides several [out-of-the-box CWS Agent rules][1] for your convenience in the default policy.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:22]:
```suggestion
An Agent Rule contains [Agent Expressions][2] that determine which activities the Agent collects. These captured events are then evaluated based on patterns of events described in a [rule][3]. 

A full set of rules for the Agent is called a policy. Datadog provides you with several [out-of-the-box CWS Agent rules][1] in the default policy.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 11]
```diff
@@ -15,23 +15,23 @@ further_reading:
 
 ## Overview
 
-With Cloud Workload Security (CWS) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of rules to detect suspicious behavior. Before learning how to create custom rules below, it is recommended to learn the basics of Cloud Workload Security rules and the concept of Agent expressions. Refer to the [Agent expression docs][1] for more information.
+With Cloud Workload Security (CWS) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of rules to detect suspicious behavior. Datadog provides [bundled CWS Agent rules][1] that you receive when you upgrade the Datadog Agent. However, you can write your own custom Agent rules. This guide covers the concept of Agent rules and provides instruction on how to create them.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:18]:
```suggestion
With Cloud Workload Security (CWS) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of rules to detect suspicious behavior. 

When you upgrade the Datadog Agent, you receive [bundled CWS Agent rules][1]. You can also write your own custom Agent rules. This guide covers Agent rules and how to create them.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 12]
```diff
@@ -69,52 +73,69 @@ For example, if you used explicit filenames to evaluate open events (for example
...
-9. Restart the [Datadog Agent][1].
-
-
-[1]: /agent/guide/agent-commands/?tab=agentv6v7#restart-the-agent
 {{% /tab %}}
 
 {{% tab "Helm" %}}
 
-8. Create a configmap containing `default.policy`. For example, `kubectl create configmap jdefaultpol --from-file=default.policy`.
+1. Create a configmap containing `default.policy`. For example, `kubectl create configmap jdefaultpol --from-file=default.policy`.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:136]:
```suggestion
1. Create a ConfigMap containing `default.policy`. For example, `kubectl create configmap jdefaultpol --from-file=default.policy`.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 13]
```diff
@@ -69,52 +73,69 @@ For example, if you used explicit filenames to evaluate open events (for example
...
 
-9. Add the configMap (`jdefaultpol`) into `values.yaml` with `datadog.securityAgent.runtime.policies.configMap`:
+2. Add the configMap (`jdefaultpol`) into `values.yaml` with `datadog.securityAgent.runtime.policies.configMap`:
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:138]:
```suggestion
2. Add the ConfigMap (`jdefaultpol`) into `values.yaml` with `datadog.securityAgent.runtime.policies.configMap`:
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 14]
```diff
@@ -134,19 +155,21 @@ Complete the remaining steps based on your environment:
         enabled: false
     ```
 
-10. Upgrade the helm chart with `helm upgrade <RELEASENAME> -f values.yaml --set datadog.apiKey=<APIKEY> datadog/datadog`.
+3. Upgrade the helm chart with `helm upgrade <RELEASENAME> -f values.yaml --set datadog.apiKey=<APIKEY> datadog/datadog`.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:158]:
```suggestion
3. Upgrade the Helm chart with `helm upgrade <RELEASENAME> -f values.yaml --set datadog.apiKey=<APIKEY> datadog/datadog`.
```

**Review by alai97 — CHANGES_REQUESTED** [review]

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 15]
```diff
@@ -53,11 +61,11 @@ To ensure that your policy is evaluated in-kernel for maximum efficiency, always
 - `[event_type].file.name`
 - `[event_type].file.path`
 
-`[event_type]` can be open or exec, for example.
+**Note**: `[event_type]` can be open or `exec`, for example.
 
 Use wildcards (`*`) carefully. For example, never use `open.file.path =~ "*/myfile"`. If you must use wildcards prefixing directories, at least two levels are required: `open.file.path =~ "*/mydir/myfile")`.
```
Comment (rishi-ilangomaran) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:66]: Can we add an explanation on why we don't recommend using wildcards - ie. can cause name conflict and ambiguities

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 16]
```diff
@@ -15,23 +15,27 @@ further_reading:
 
 ## Overview
 
-With Cloud Workload Security (CWS) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of rules to detect suspicious behavior. Before learning how to create custom rules below, it is recommended to learn the basics of Cloud Workload Security rules and the concept of Agent expressions. Refer to the [Agent expression docs][1] for more information.
+With Cloud Workload Security (CWS) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of rules to detect suspicious behavior. 
 
-The full set of rules for the Agent is called a policy. Datadog provides several [out-of-the-box Cloud Workload Security rules][2] for your convenience in the default policy.
+When you upgrade the Datadog Agent, you receive [bundled CWS Agent rules][1]. You can also write your own custom Agent rules. This guide covers Agent rules and how to create them.
```
Comment (rishi-ilangomaran) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:20]: When you upgrade the Datadog Agent, you receive bundled CWS Agent rules used in our [default Signal Rules ](https://docs-staging.datadoghq.com/sarina/cws-rule-split/security_platform/default_rules/#cat-workload-security). You can also write your own custom Agent rules. This guide covers Agent rules, how to create them, and how to use them to generate security signals.

Comment (rishi-ilangomaran) [reply content/en/security_platform/cloud_workload_security/workload_security_rules.md:20]: The URL is changed to filter by CWS rules

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 17]
```diff
@@ -15,23 +15,27 @@ further_reading:
...
 
-At some point, you will likely want to write your own custom rules for the Agent to use. Below are example rules, guidelines for optimization, and the underlying mechanics of rule efficiency.
+### Agent rules
 
-## Rule expressions
+An Agent Rule contains [Agent Expressions][2] that determine which activities the Agent collects. These captured events are then evaluated based on patterns of events described in a [rule][3]. 
 
-Rule expressions define behavior based on activity in your hosts and containers. For example, if you want to detect the following behavior, “the passwd command executed”, there are a few attributes to note.
+A full set of rules for the Agent is called a policy. Datadog provides you with several [out-of-the-box CWS Agent rules][1] in the default policy.
```
Comment (rishi-ilangomaran) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:26]: A full set of rules for the Agent is called a policy. Datadog provides you with several [out-of-the-box CWS Agent](https://docs-staging.datadoghq.com/sarina/cws-rule-split/security_platform/default_rules/#cat-workload-security) Rules powered by the default Agent policy

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md (hunk 16 above)
Comment (sarina-dd) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:20]:
```suggestion
When you upgrade the Datadog Agent you receive bundled CWS Agent rules, which are used in the [default signal rules][1]. You can also write your own custom Agent rules. This guide covers Agent rules, how to create them, and how to use them to generate security signals.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 18]
```diff
@@ -134,19 +159,21 @@ Complete the remaining steps based on your environment:
...
 
 ## Further Reading
 {{< partial name="whats-next/whats-next.html" >}}
 
-[1]: /security_platform/cloud_workload_security/agent_expressions
-[2]: /security_platform/default_rules
-[3]: /agent/
-[4]: https://app.datadoghq.com/security/configuration/rules
-[5]: /security_platform/cloud_workload_security/agent_expressions/
+[1]: /security_platform/default_rules
```
Comment (sarina-dd) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:175]:
```suggestion
[1]: /security_platform/default_rules/#cat-workload-security
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md (hunk 17 above)
Comment (sarina-dd) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:26]:
```suggestion
A full set of rules for the Agent is called a policy. Datadog provides you with several [out-of-the-box CWS Agent rules][1] powered by the default Agent policy.
```

**Comment by sarina-dd** [comment]
@rishi-ilangomaran Updated based on the notes, thanks for adding! If there are anymore edits, let me know. If not, we can push to final review + merge.

**Review by alai97 — APPROVED** [review]

**Review by rishi-ilangomaran — APPROVED** [review]
LGTM - can this be merged as early as possible on Tuesday 25th, 2021

**Pull request merged**